doc/contributing/docs/markup.rst

```diff
@@ -143,27 +189,28 @@ The tag can be anything meaningful. The only guideline is for Tarantool syntax
 items (such as members), where the preferred tag syntax is
 ``module_or_object_name dash member_name``. For example, ``box_space-drop``.
 
-------------------------------
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Linking to external resources
-------------------------------
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Avoid separating the link and the target definition, like this:
+To make an external link, use the following syntax:
 
-..  code-block:: rst
+..  code-block:: text
 
-    This is a paragraph that contains `a link`_.
+    This is a paragraph that contains `a link <http://example.com/>`_.
 
-    ..  _a link: http://example.com/
+Avoid separating the link and the target definition, like this:
 
-Use non-separated links instead:
+..  container:: dont
```

Comment:
@Onvember special class for highlighting bad examples. No styles yet, will add them in the future.